pkgs/tools/networking/imapsync/default.nix

```
@@ -1,35 +1,67 @@
-{lib, stdenv, makeWrapper, fetchurl, perl, openssl, perlPackages }:
+{ lib, stdenv, makeWrapper, fetchFromGitHub, perl, openssl, perlPackages, procps }:
 
 stdenv.mkDerivation rec {
   pname = "imapsync";
-  version = "1.727";
-
-  src = fetchurl {
-    url = "https://releases.pagure.org/imapsync/imapsync-${version}.tgz";
-    sha256 = "1axacjw2wyaphczfw3kfmi5cl83fyr8nb207nks40fxkbs8q5dlr";
+  version = "2.140";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "${pname}-${version}";
```

```suggestion
    rev = "imapsync-${version}";
```
Substituting pname a lot makes the code harder to understand.